spec/controllers/admin/languages_controller_non_admin_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+#This file tests that non logged in users and non admin schools can not perform admin compromised actions
```

Comment:
no need for the comments, the describe, context and it blocks should be taking care of that.